config/routes.rb

```diff
@@ -136,8 +136,10 @@
 
   match "/oembed", to: 'master_files#oembed', via: [:get]
   
-  require 'resque/server'
-  mount Resque::Server, at: '/jobs'
+  authenticate :user, lambda {|u| Ability.new(u).can? :manage, Resque } do
```

Comment:
@bmquinn and I came up with a good way to use this using `constraints`. Please see #1351 for an example that can be reused easily by this issue.